tsl/test/sql/dist_ddl.sql

```diff
@@ -150,6 +145,40 @@ INSERT INTO disttable VALUES
 
 SELECT * FROM show_chunks('disttable');
 
+-- Rename column
+ALTER TABLE disttable RENAME COLUMN description TO descr;
+SELECT * FROM test.show_columns('disttable')
+WHERE "Column"='descr';
+
+SELECT * FROM test.remote_exec('{ data_node_1 }', $$
+	   SELECT chunk.relid AS chunk_relid,
+	   		  (SELECT "Column" AS col FROM test.show_columns(chunk.relid) WHERE "Column"='descr')
+	   FROM (SELECT "Child" AS relid FROM test.show_subtables('disttable') LIMIT 1) chunk
+$$);
+
+-- Rename constraint
+ALTER TABLE disttable ADD CONSTRAINT device_check CHECK (device > 0);
+ALTER TABLE disttable RENAME CONSTRAINT device_check TO device_chk;
+SELECT * FROM test.show_constraints('disttable')
+WHERE "Constraint"='device_chk';
+
+SELECT * FROM test.remote_exec('{ data_node_1 }', $$
+       SELECT chunk.relid AS chunk_relid,
+	   		  (SELECT "Constraint" AS constr FROM test.show_constraints(chunk.relid) WHERE "Constraint"='device_chk')
+	   FROM (SELECT "Child" AS relid FROM test.show_subtables('disttable') LIMIT 1) chunk
+$$);
```

Comment:
You have added tests that add constraints to tables as well as renaming columns, but I see no mention of this in the commit message and it does not seem to be any changes in the implementation. Could you add a description about why these were added to the commit message?

Reply:
Clarified in PR description.